### PR TITLE
Hacktoberfest: Add solution to problem 74

### PR DESCRIPTION
**Pull request by GGn0**
### **Describe your change:**
Add a solution to project Euler problem 74

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

#2695

### CODE REVIEW

**Review by poyea — COMMENTED** [review]
Please read the dedicated README for `project_euler`. In particular, the directory you need to put the algorithm in and some function signatures. 

https://github.com/TheAlgorithms/Python/blob/master/project_euler/README.md

File: project_euler/problem_74/sol1.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+    return fact_sum
+
+
+def solution() -> int:
+    """Returns the number of numbers that produce
+        chains with exactly 60 non repeating elements.
+    >>> solution()
+    402
+    """
```
Comment (poyea) [inline project_euler/problem_74/sol1.py:83]: You'll have to make a default parameter for solution.

File: project_euler/problem_74/sol1.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+        """ If the while exited because the chain list contains the exact amount of elements
+            increase the counter
+        """
+        chain_counter += len(chain_list) == EXACT_CHAIN_LENGTH
+
+    return chain_counter
+
+
+print(solution())
```
Comment (poyea) [inline project_euler/problem_74/sol1.py:114]: Put this into something like:
```python
if __name__ == "__main__":
    import doctest

    doctest.testmod()
    print(solution())

**Review by poyea — APPROVED** [review]

**Comment by poyea** [comment]
Thank you for your pull request!🤩

**Comment by dhruvmanila** [comment]
This solution needed optimization as its taking 2 minutes to run. Now our tests will take 2x longer to finish lol.

**Pull request merged**